src/api/routes/youtube.js

```diff
@@ -0,0 +1,34 @@
+import { Router } from 'express';
+import { logger } from '../../utils/logger';
+import APICall from '../../utils/api/calls';
+import { CACHE_TIME } from '../../utils/config';
+import { getCache, setCache } from '../../utils/cacheData';
+
+const router = Router();
+const key = 'youtube';
+
+router.get('/', async (req, res) => {
+	const cacheContent = getCache(key);
+	if (cacheContent) {
+		logger.info('Videos sent from cache');
+		return res.status(200).json(cacheContent);
+	}
```

Comment:
Make sure you write test for these lines: https://coveralls.io/builds/42313127/source?filename=src%2Fapi%2Froutes%2Fyoutube.js#L13